rmi/servercode/ResImpl/MiddleWareImpl.java

```diff
@@ -14,15 +14,14 @@
 
 public class MiddleWareImpl implements ResourceManager
 {
-
+    public static int RM_Index = 0;
```

Comment:
This is pretty much just a banner. 0 for the MW, 1 for the Car RM, 2 for the Hotel RM, 3 for the Flight RM.
It's hardcoded a bit but whatevs it's only 4 RM's